webapp/components/Form.tsx

```diff
@@ -46,6 +53,18 @@ const Form = (): JSX.Element => {
     images,
   };
 
+  const fillUserInfo = (name: string, mailFrom: string, accountNumber: string, token: string) => {
+    setName(name)
+    setMailfrom(mailFrom)
+    setAccountNumber(accountNumber)
+    setToken(token)
+  }
+
+  const updateGroup = (group: string) => {
+    setGroup(group)
+    setMailto(`${group.toLowerCase()}-kasserer@ntnui.no`)
```

Comment:
This doesn't work correctly for group names with spaces, such as Ski og Fjellsport would return `ski og fjellsport-kasserer@ntnui.no`, and gsuite_prefix might be customized!

Reply:
I'll leave the gsuite prefix stuff for later as I'm not sure how I pass the group reference around in the <Autocomplete> component